.github/workflows/ci.yml

```diff
@@ -147,87 +155,95 @@ jobs:
 
     services:
       zookeeper:
-        image: wurstmeister/zookeeper
+        image: bitnami/zookeeper:latest
         ports:
           - "2181:2181"
+        env:
+          ALLOW_ANONYMOUS_LOGIN: yes
 
       kafka1:
-        image: wurstmeister/kafka:2.13-2.7.1
```

Comment:
This test block is titled test271 and is supposed to test Kafka version 2.7.1

Reply:
This has been fixed in the latest commit